hathor/utils/weight.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import math
+
+# _LN_2 = math.log(2)
+
+
+def weight_to_work(weight: float) -> int:
+    """Convert weight to work rounding to the nearest integer."""
+    # return int(0.5 + 2**weight)
```

Comment:
I guess we should remove the commented lines because I guess it makes no sense to change it in the future.

Reply:
Done.

Reply:
Thanks!